docs/source/developer_guide/issue-management.md

```diff
@@ -0,0 +1,118 @@
+<!--
+{% comment %}
+Copyright 2018-2021 Elyra Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+## Issues Workflow
+This section describes how Elyra issues are organized. 
+
+### Label and issue tagging
+
+
+|  | Tags |
+| --- | --- |
+| Problems | ![][bug] ![][security] ![][production] |
+| Experience | ![][copy] ![][design] ![][ux] |
+| Feedback | ![][discussion] ![][rfc] ![][question] |
+| Improvements | ![][enhancement] ![][optimization] |
+| Area | ![][backend] ![][community] ![][documentation] ![][front-end] |
+| Satus | ![][needs discussion] ![][needs review]  ![][needs triage] ![][needs update] ![][waiting for author] ![][work in progress]|
+| Impact | ![][blocker] ![][bocked]  ![][do not merge]|
```

Comment:
```suggestion
| Status | ![][needs discussion] ![][needs review]  ![][needs triage] ![][needs update] ![][waiting for author] ![][work in progress] |
| Impact | ![][blocker] ![][bocked]  ![][do not merge] |
```

Reply:
There's also a typo in `Impact`: `bocked` -> `blocked`